wallet/src/main/java/ch/admin/bag/covidcertificate/wallet/transfercode/worker/TransferWorker.kt

```diff
@@ -84,7 +84,7 @@ class TransferWorker(context: Context, workerParams: WorkerParameters) : Corouti
 	}
 
 	private suspend fun downloadTransferCertificate(transferCode: TransferCodeModel): TransferState {
-		val keyPair = TransferCodeCrypto.loadKeyPair(transferCode.code)
+		val keyPair = TransferCodeCrypto.loadKeyPair(transferCode.code, context)
```

Comment:
Wouldn't applicationContext work?
(Same on 108)

Then the context wouldn't need to be persisted in a member variable.

Reply:
True, I didn't see that the applicationContext was available in the worker